scripts/deploy.sh

```
@@ -0,0 +1,76 @@
+    local username=$(yq '.spring.datasource.username' "$file")
+    local password=$(yq '.spring.datasource.password' "$file")
+
+    local root_password=$(yq '.spring.datasource.root-password // .spring.datasource.password' "$file")
+
+    echo "${prefix}_DB_USERNAME=$username"
+    echo "${prefix}_DB_PASSWORD=$password"
+    echo "${prefix}_DB_ROOT_PASSWORD=$root_password"
+}
+
+echo "==> Generating .env from secret submodule..."
+> "$PROJECT_ROOT/.env"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)